# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/reftest.list
# Commit: de68ee31172f
# Full Hash: de68ee31172fbe2d5c4a34b7ca6f4f659433461e
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2025-05-07 04:19:47
# Regressor Bug: 1961387
# File Overlap Count: 3
# Description:
#   Bug 1961387 - Simple reftest for Noto Color Emoji in vertical mode. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D248079
# ==============================================================================

diff -r a4ad6f718272 -r de68ee31172f gfx/tests/reftest/reftest.list
--- a/gfx/tests/reftest/reftest.list	Tue May 06 20:12:45 2025 +0000
+++ b/gfx/tests/reftest/reftest.list	Tue May 06 20:12:46 2025 +0000
@@ -48,4 +48,5 @@
 != 1870240-colrv1-cycle.html 1870240-colrv1-cycle-notref.html
 == 1909933.html 1909933-ref.html
 fuzzy(0-2,0-6229) random-if(Android) == 1912431-emoji-globalAlpha.html 1912431-emoji-globalAlpha-ref.html
+== 1961387-1.html 1961387-1-ref.html
 skip-if(!cocoaWidget) == 1964225-1.html 1964225-1-ref.html
